deb/deb.go

```diff
@@ -166,7 +165,7 @@ func (*Deb) SetPackagerDefaults(info *nfpm.Info) {
 	// if in the long run we should be more strict about this and error when
 	// not set?
 	if info.Maintainer == "" {
-		log.Println("DEPRECATION WARNING: Leaving the 'maintainer' field unset will not be allowed in a future version")
```

Comment:
We don't use `log`, elsewhere, so I changed it to print to `os.Stderr` just like the `empty_folders` deprecation warning.

Reply:
thinking out loud here: maybe we could allow the caller to pass a `io.Writer` or some other interface as to "where should we print the warnings" 🤔

Reply:
Another idea, a package level `io.Writer` for warnings that defaults to `os.Stderr` that can be overwritten globally. It's simple enough to add it to this PR.